media/videostitcher/create_live_session.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package videostitcher
+
+// [START video_stitcher_create_live_session]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	stitcherpb "google.golang.org/genproto/googleapis/cloud/video/stitcher/v1"
+)
+
+// createLiveSession creates a livestream session in which to insert ads.
+// Live sessions are ephemeral resources that expire after a few minutes.
+func createLiveSession(w io.Writer, projectID, sourceURI, adTagURI, slateID string) error {
+	// projectID := "my-project-id"
+
+	// Uri of the media to stitch; this URI must reference either an MPEG-DASH
+	// manifest (.mpd) file or an M3U playlist manifest (.m3u8) file.
+	// sourceURI := "https://storage.googleapis.com/my-bucket/main.mpd"
+
+	// See Single Inline Linear (https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/tags)
```

Comment:
Are there actual docs describing the API that could be linked to rather than the list of examples? Seems confusing. Also, this could probably be a local variable rather than an argument; I don't think it needs to be subbed in for testing.

Reply:
Added a link to some conceptual documents. Made the ad tag URL a local variable.